web/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:alpine
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app
+
+RUN npm install -g @angular/cli
+
+RUN npm install -g pnpm
+
+RUN pnpm install
+
+CMD ["ng", "serve", "--host", "0.0.0.0"]
```

I will change the order of instructions in order to be efficient when modifications have been done. Because, currently, if you update your codebase, you'll have to reinstall angular cli and pnpm each fucking time. So it's quite annoying...
```suggestion
FROM node:alpine

RUN npm install -g @angular/cli

RUN npm install -g pnpm

WORKDIR /usr/src/app

COPY package.json .
COPY pnpm-lock.yaml .

RUN pnpm install

COPY . .

CMD ["ng", "serve", "--host", "0.0.0.0"]
```